src/components/search/History/SearchHistory.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Icon as UIIcon, List as UIList } from '@faststore/ui'
+import Button from 'src/components/ui/Button'
+import './SearchHistory.scss'
+import Link from 'src/components/ui/Link'
+import Icon from 'src/components/ui/Icon'
+import useSearchHistory from 'src/sdk/search/useSeachHistory'
+import { formatSearchState, initSearchState } from '@faststore/sdk'
+
+const doSearch = (term: string) => {
+  const { pathname, search } = formatSearchState(
+    initSearchState({
+      term,
+      base: '/s',
+    })
+  )
+
+  return `${pathname}${search}`
+}
+
+interface SearchHistoryProps {
+  onClear: () => void
+}
+
+const SearchHistory = ({ onClear }: SearchHistoryProps) => {
+  const { searchHistory } = useSearchHistory()
+
+  return (
+    <section data-store-search-history className="history__section">
+      <div className="history__header">
+        <p className="history__title">History</p>
+        <Button variant="tertiary" onClick={onClear}>
+          Clear
+        </Button>
+      </div>
+      <UIList variant="ordered">
+        {searchHistory.map((item, index) => (
+          <li key={index}>
+            <Link variant="display" to={doSearch(item)}>
+              <div>
+                <UIIcon
+                  component={<Icon name="Clock" width={18} height={18} />}
+                />
```

Comment:
Is this the same as the following?
```suggestion
                <UIIcon name="Clock" width={18} height={18} />
```

Same comment for the `ArrowUpRight` below.

If it is, then maybe we can use this shorter version.

Reply:
It's not. The icons are being called like that because of these PR's:

-  [SVG unification](https://github.com/vtex-sites/base.store/pull/378)
- [Renaming IconSVG to Icon](https://github.com/vtex-sites/base.store/pull/386)